lib/protocol/src/openctx-protocol.schema.json

```diff
@@ -105,31 +114,58 @@
       "properties": {},
       "tsType": "Record<string, never>"
     },
+    "MentionSelector": {
+      "description": "List of regex patterns matching the mention text for which the provider can return mentions.",
+      "type": "object",
+      "additionalProperties": false,
+      "required": ["pattern"],
+      "properties": {
+        "pattern": {
+          "description": "The regex pattern matching the mention text for which the provider can return mentions",
+          "type": "string"
+        }
+      }
+    },
+    "MentionKind": {
+      "type": "object",
+      "description": "A mention kind supported by the provider.",
+      "additionalProperties": false,
+      "required": ["id", "title"],
```

Comment:
title should not be required (simplifies mock/test code)

Reply:
Removed mention kind.